docs/_writing_parameters.rst

```diff
@@ -77,7 +77,7 @@ support `Cheetah <http://www.cheetahtemplate.org/docs/users_guide_html/users_gui
 ::
 
     #if $mask_regions
-    -M $mask_regions
+    -M "$mask_regions"
```

Comment:
BTW, it seems these docs are duplicated... http://planemo.readthedocs.io/en/latest/standards/docs/best_practices/tool_xml.html and https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html Is there a migration taking place?